### DISCUSSION TITLE
Store links

### DISCUSSION THREAD

**Discussion by TwoPigsToTheWind** [slu4coder/Minimal-64x4-Home-Computer#20] [Q&A] [answered]
Does anyone have a link or item number handy for the reset switch and oscillator?

**Comment by hans61** [comment, marked as answer]
16 MHz DIP-14 quartz oscillators are hard to find. In which country are you looking for a dealer?
I also tested the Minimal 64x4 with 15.977 and 16.258 MHz oscillators and found no problems.
I was hoping to get a picture on monitors that didn't have one before. But everything stayed the same.
The reset button should be easy to find as a short-stroke 6x6 button.
e.g.

https://www.mouser.de/ProductDetail/CUI-Devices/TS02-66-153-BK-160-LCR-D?qs=A6eO%252BMLsxmQ5pPFnT%2FC5Lw%3D%3D&mgh=1&vip=1&utm_id=20979042634&gad_source=1&gclid=Cj0KCQjwvb-zBhCmARIsAAfUI2umzOLATlNZyhIgxCbiL8oPUr09Z93JjJmIA2ez2lgsQ45g8p-2DmQaAgB6EALw_wcB

**Comment by joelburton** [comment]
I found my oscillator at https://www.digikey.com/en/products/detail/abracon-llc/ACO-16-000MHZ-EK/675402

(the switches are the super-common kind ; I used one from https://www.amazon.com/TWTADE-Yellow-Orange-6x6x5mm-Tactile/dp/B07C7211PJ/

**Comment by joelburton** [comment]
... and I used the red button so the computer would run faster ;-)

**Comment by TwoPigsToTheWind** [comment]
Thanks all! Very nearly done putting it together. Still waiting on a few pieces to arrive but everything has been ordered. 
![minimal_progress](https://github.com/slu4coder/Minimal-64x4-Home-Computer/assets/169104967/0fd43b8c-5da3-457c-aaf8-43f6bf0b4c35)

**Reply by joelburton** [reply]
@TwoPigsToTheWind : it might just be that I'm seeing the shadow wrong, but it looks like you might have U16 and U17 incorrectly placed --- those should have their top pin on the right, not the the left.

**Reply by TwoPigsToTheWind** [reply]
good eye, they were both backwards, good thing all the ICs are socketed
